src/redfish_collector/core/rawCollector.py

```
@@ -222,9 +251,23 @@ async def dataCollector(serverAddress,username,password,templateDir,logLevel):
     dataRaw = dict()
+    try:
+        timeout = ClientTimeout(total=60)
+        async with ClientSession(timeout=timeout) as session:
+            async with session.delete(logoutURL, headers={'X-Auth-Token': tokenValue}, ssl=False) as response:
+                if response.status == 200 or response.status == 204:
+                    logging.info("[%s] Logged out successfully" % serverAddress)
+                    pass
+                else:
+                    logging.error("[%s] Logout failed with status code: %s" % (serverAddress,response.status))
+            pass
+    except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)